shell/platform/android/io/flutter/view/AccessibilityBridge.java

```diff
@@ -229,15 +250,25 @@ public AccessibilityNodeInfo createAccessibilityNodeInfo(int virtualViewId) {
                 result.addAction(AccessibilityNodeInfo.ACTION_SCROLL_BACKWARD);
             }
         }
-
+        if (object.hasFlag(Flag.IS_LIVE_REGION)) {
+            result.setLiveRegion(View.ACCESSIBILITY_LIVE_REGION_POLITE);
+        } else if (object.hadFlag(Flag.IS_LIVE_REGION)) {
+            mLiveRegions.remove(object.id);
+        }
+        
         boolean hasCheckedState = object.hasFlag(Flag.HAS_CHECKED_STATE);
-        result.setCheckable(hasCheckedState);
+        boolean hasToggledState = object.hasFlag(Flag.HAS_TOGGLED_STATE);
+        assert !(hasCheckedState && hasToggledState);
```

Comment:
Can we enforce this on the framework side? This should also be documented on the flags that the two are mutually exclusive.

Reply:
Add document, yes I will make sure we have an assert too